src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -2,6 +2,32 @@ import React from 'react';
 
 import './CommentInfo.scss';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type Props = {
+  name:string,
+  email: string,
+  body: string,
+};
+
+export const CommentInfo: React.FC<Props> = ({ name, email, body }) => (
+  <div
+    className="commentInfo"
+  >
+    <div
+      data-cy="comment-name"
+    >
+      {name}
+    </div>
```

Comment:
it indeed looks prettier
```suggestion
    <div data-cy="comment-name">
      {name}
    </div>
```